WTWSMS/common/minor_titles/00_minor_titles.txt

```diff
@@ -2421,28 +2368,30 @@ title_yabgu = {
 
 	allowed_to_grant = {
 		#OR = {
-		#	AND = {
+			AND = {
 				is_nomadic = yes
 				NOT = { culture = mongol }
-		#	}
-		#	AND = {
-		#		OR = {
-		#			is_feudal = yes
-		#			is_tribal = yes
-		#		}
-		#		NOT = {
-		#			OR = {
-		#				has_landed_title = e_mongol_empire
-		#				has_landed_title = e_golden_horde
-		#				has_landed_title = e_il-khanate
-		#			}
-		#		}
-		#		OR = {
-		#			religion = tengri_pagan
-		#			religion = tengri_pagan_reformed
-		#		}
-		#	}
-		#}
+			}
+			AND = {
+				OR = {
+					is_feudal = yes
+					is_tribal = yes
+					is_nomadic = yes
+				}
+				NOT = {
+					OR = {
+						has_landed_title = e_mongol_empire
+						has_landed_title = e_golden_horde
+						has_landed_title = e_il-khanate
+					}
+				}
+				OR = {
+					is_nomadic = yes
+					religion = tengri_pagan
+					religion = tengri_pagan_reformed
+				}
+			}
+		}
```

Comment:
Are you really going to say that handing out a Minor Title is ahistorical, but the entire Byzantine Empire going Hellenic, an Egyptian pantheon worshipping Hunnic Empire constructing pyramids. or Assyrian Paganism being ressurected, is acceptable?